pipelines/common_mce_2.9.yaml

```diff
@@ -55,7 +55,7 @@ spec:
       description: Add built image into an OCI image index
       name: build-image-index
       type: string
-    - default: []
+    - default: [MCE_VERSION="v2.9.1"]
```

Comment:
Maybe there will be an `ACM_VERSION` in the future?

Reply:
Updated to avoid changing the `build-args`.